components/textbox/src/index.scss

```diff
@@ -48,7 +48,8 @@
   @include utrecht-textbox--postal-code-nl-size;
 }
 
-.utrecht-textbox--house-number-size {
+.utrecht-textbox--house-number-size,
+.utrecht-textbox[name="huisnummer"] {
```

Comment:
`[name="huisnummer"]` hoort niet thuis in deze algemene CSS componente, die zou in de OpenFormsContainer moeten